pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,144 @@
+{ config
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, stdenv
+, bash
+, intltool
+, glib
+, glibcLocales
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, procps
+, wrapGAppsHook
+, yelp
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  # Uncomment for tests
+  #LC_ALL = "en_US.UTF-8";
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    python3
+    python3.pkgs.pycairo
+    python3.pkgs.dbus-python
+    python3.pkgs.pygobject3
+    python3.pkgs.systemd
+    python3.pkgs.distutils_extra
+    python3.pkgs.pyatspi
+    glib
+  ];
+
+  buildInputs = [
+    #python3.pkgs.nose # Needed for testing
+    gtk3
+    libcanberra_gtk3
+    libudev
+    bash
+    hunspell
+    isocodes
+    pkgconfig
+    xorg.libXtst
+    xorg.libxkbfile
+    libxkbcommon
+    intltool
+    glibcLocales
```

Comment:
`pkgconfig`, `intltool` and `glibcLocales` are only needed for build itself (not linked or otherwise used in the final product) so they should be moved to `nativeBuildInputs`.